tensorboard/components/tf_storage/storage.ts

```diff
@@ -184,13 +213,12 @@ function makeBindings<T>(fromString: (string) => T, toString: (T) => string): {
       // if this is a localStorage property, we always synchronize the value
       // in localStorage to match the one currently in the URI.
       if (fullOptions.useLocalStorage) {
-        set(uriStorageName, newVal, true);
-      }
-      if (!_.isEqual(newVal, get(uriStorageName, false))) {
+        set(uriStorageName, newVal, fullOptions);
+      } else if (!_.isEqual(newVal, get(uriStorageName, {useLocalStorage: false}))) {
```

Comment:
Now that this is an `else if` we're no longer always reflecting observed property changes to the URI (as desired).  That means we should update the commentary up in getInitializer() that says "the corresponding observer will proxy the localStorage value into URI storage" since it's no longer true.